docs/ruleset_functions/get_zone_eflh.md

```diff
@@ -0,0 +1,43 @@
+
+**Inputs:** 
+- **RMI**
+- **zone**
+
+**Returns:**  
+- **flh**: a number equal to the total equivalent full load hours for the year
+ 
+**Function Call:**
+- **get_list_hvac_systems_associated_with_zone**
+
+## Logic:
+- create a dictionary to hold information about the occupancy of each space: `people_info_for_spaces = {}`
+- create a variable for the total number of zone occupants: `total_zone_occupants = 0`
+- loop through the spaces in the zone collecting information about occupancy: `for space in zone.spaces:`
+  - append the information about the occupancy to the list: `people_info_for_spaces[space]["NUM_OCCUPANTS] = space.number_of_occupants`
+  - `people_info_for_spaces[space]["SCHEDULE_HOURLY_VALUES"] = space.occupant_multiplier_schedule.hourly_values`
+  - add the occupants of this space to the total number of zone occupants: `total_zone_occupants += space.number_of_occupants`
+- get the list of the hvac systems associated with the zone: `hvac_systems_list = get_list_hvac_systems_associated_with_zone(RMI,zone.id)`
+- create an index to which each full load hour will be added: `flh = 0`
+- now create a variable num_hours_in_year which will take the number of hours in the schedule for the first (really any) space in people_info_for_spaces: `num_hours_in_year = len(people_info_for_spaces[list(people_info_for_spaces.keys())[0]]["SCHEDULE_HOURLY_VALUES"])`
+- now calculate flh for each hour of the year: `for hour in range(num_hours_in_year):`
+  - calculate the number of people in the zone this hour: `occupants_this_hour = 0`
+  - loop through the people_info_for_spaces: `for space in people_info_for_spaces:`
+    - multiply the max number of people in this space by the schedule value and add it to people_this_hour: `occupants_this_hour += space_people_info[space]["NUM_OCCUPANTS"] * space_people_info[space]["SCHEDULE_HOURLY_VALUES"][hour]`
+  - check if the people this hour is greater than 5%: `if((occupants_this_hour / total_zone_occupants)>0.05:`
+    - now check to see if there are any HVAC systems that are operational this hour.  Create a boolean: `hvac_systems_operational_this_hour = FALSE`
+    - loop through the hvac_systems_list: `for hvac_system in hvac_systems_list:`
+      - check if the HVAC system fan operating schedule is greater than 1 this hour: `if hvac_system.fan_system.operation_schedule[hour] == 1:`
+        - the system is operational this hour, set the boolean to TRUE: `hvac_systems_operational_this_hour = TRUE`
+    - check if the boolean is true: `if hvac_systems_operational_this_hour:`
+      - add the hour to the flh: `flh += 1`
+
+**Returns** `flh`
+
+**Questions:**
+# 1.  Do we need to do a check to see if FanSystem and it's operating schedule exist before calling hvac_system.fan_system.operation_schedule?
```

Comment:
Yes, we need to check for it I think, because if it's missing than the system is continuously on based on the note for the FanSystem.operating_schedule